kedro-datasets/kedro_datasets/pandas/sql_dataset.py

```diff
@@ -275,6 +283,25 @@ def _exists(self) -> bool:
         schema = self._load_args.get("schema", None)
         return insp.has_table(self._load_args["table_name"], schema)
 
+    def preview(self, nrows: int = 5) -> TablePreview:
+        """
+        Generate a preview of the dataset with a specified number of rows.
+
+        Args:
+            nrows: The number of rows to include in the preview. Defaults to 5.
+
+        Returns:
+            dict: A dictionary containing the data in a split format.
+        """
+
+        table_name = _sanitise_table_name(self._load_args["table_name"])
+
+        sql_query = f"SELECT * FROM {table_name} LIMIT {nrows}"  # nosec
```

Comment:
is `LIMIT` universal to all SQL dataset? I remember SQL SERVER use `TOP` instead.

https://stackoverflow.com/questions/1891789/sql-select-first-10-rows-only